src/Symfony/Component/Serializer/Normalizer/ProblemNormalizer.php

```diff
@@ -53,15 +60,33 @@ public function normalize(mixed $object, string $format = null, array $context =
             throw new InvalidArgumentException(sprintf('The object must implement "%s".', FlattenException::class));
         }
 
+        $data = [];
+        $exception = $context['exception'] ?? null;
+        if ($exception instanceof HttpExceptionInterface) {
+            $exception = $exception->getPrevious();
+
+            if ($exception instanceof PartialDenormalizationException) {
+                $data = [
+                    self::TYPE => 'https://symfony.com/errors/denormalization',
+                    'errors' => array_map(fn ($e) => ['title' => $e->getMessage()], $exception->getErrors()),
```

Comment:
I would be nice to add some hint if possible!

```
if ($exception->canUseMessageForUser()) {
    $parameters['hint'] = $exception->getMessage();
}
```

Reply:
updated